mount tool

BehrouzSohrabi opened the pull request:
Mount tool added.

BehrouzSohrabi committed:
Create mount.md

shabane commented:
سلام، بابت فصل جدیدی که اضافه کردین ممنونم.
بابت تاخیر من خیلی ببخشید.
چند نکته هست که نیاز هست بدونین.

- اگر ممکنه کلمه "نحو" رو با "ساختار دستور" عوض کنید. نحو یکم گنگ هست.
- قسمت های option و type و سوییچ ها رو از هم باید جدا کرد. در واقع باید قسمتی که type رو توضیح میده از قسمتی که option ها رو توضیح میده جدا باشه. و همچین هر سوییچ توی بخش مربوط به وظیفه ان سوییچ توضیح داده بشه.

مثلا اینطوری ممکنه بهتره باشه:

تعیین نوع فایل سیستم:
-t type
....
برخی از انواع فایل سیستم ها عبارت اند از ext4, exfat, fat, ntfs, btrfs...

---

تعیین گزینه های مانت:
توضیحات.....

 برخی از گزینه های رایج عبارتند از:
- rw توضیحات
- ro توضیحات

---

برای مثال یک سوییچ اینطوری باید باشه:


### پرینت لاگ در خروجی
برای اینکه لاگ های برنامه mount را در خروجی ببینید از سوییچ v- استفاده کنید، با این سوییچ خطا و اخطار ها را مشاهده خواهید کرد.....


- و اینکه در آخر اگر یک سوییچ رو توضیح میدی صرفا فقط نباید نوشته بشه این سویچ فلان کار رو میکنه،
مثلا سوییچ bind-- قابل درک نبود برای من، اگر با man page مقایسه کنیم، man page خیلی کامل این سوییچ رو توضیح داده و مثال زده.


ممنونم بابت وقتی که میذاری :)

BehrouzSohrabi committed:
syntax edited

BehrouzSohrabi commented:
ممنون از کار خوبتون
بزودی اعمال میکنم

BehrouzSohrabi committed:
update mount

هر سوییچ بصورت جدا بهمراه مثال توضیح داده شد
سوییچ -o بهمراه گزینه های کاربردی عمومی آورده شد.

BehrouzSohrabi commented:
سلام لطفا کامیت آخر رو ملاحظه کنید.

shabane commented:
> سلام لطفا کامیت آخر رو ملاحظه کنید.

سلام، حتما

shabane commented:
hello, this chapter is merged with wrong branch :/
can you create new pull request with same content?

BehrouzSohrabi commented:
Hi, Sure.
